packages/playground/src/components/pull-request/pull-request-commits.tsx

```diff
@@ -1,9 +1,77 @@
-import React from 'react'
+import React, { useMemo } from 'react'
```

Comment:
If pull request commits is a component and not a page in itself, it should receive data from it's parent page.

path : `packages/src/components`

Reply:
packages/playground/src/pages/pull-request-commits-page.tsx
packages/playground/src/components/pull-request/pull-request-commits.tsx - the above page PullRequestCommitsPage calls on this so should i put it in above file because i thought i did it similiarily for the conversation page
packages/playground/src/components/pull-request/pull-request-conversation.tsx
@vardanbansal-harness

Reply:
![image](https://github.com/user-attachments/assets/1fb1e408-8f8f-4f5f-9421-c4ff0b9c5402)
this is the file in above pic^

Reply:
Should be okay I guess, these are individual tabs as sort of pages itself, individual api calls for each of tabs can be made individually by them.